packages/block-editor/src/components/iframe/index.js

```diff
@@ -285,14 +291,40 @@ function Iframe(
 				{ iframeDocument &&
 					createPortal(
 						<>
-							<head ref={ headRef }>{ head }</head>
+							<head ref={ headRef }>
+								{ head }
+								<style>
+									{ `html { transition: background 5s; ${
```

Comment:
@jasmussen I was a bit sceptical to always change the color in `html` of `iframe` component, so I came up with this hack with transition duration. What do you think? The alternative is always apply the color..

Reply:
Visually this looks good to me, so technically I'll trust you on it:
![zoom](https://user-images.githubusercontent.com/1204802/186625914-8d31d9d2-69b4-4d1c-8fd4-52832b4ae247.gif)

Structurally you can think of the dark background as a material that's always there, at the very bottom of our user interface. The topbar and sidebar and editor are laid out on top of it. When we need to zoom out the editor (thumbnail view, focus mode, zoom out view, or otherwise), this dark bottom material is shown.